src/social-network-provider/twitter.com/utils/fetch.ts

```diff
@@ -183,7 +183,7 @@ export const postImageParser = async (node: HTMLElement) => {
                         const content = await Services.Steganography.decodeImageUrl(url, {
                             pass: posterIdentity.toText(),
                         })
-                        return /https:\/\/.+\..+\/%20(.+)%40/.test(content) ? content : ''
+                        return /https:\/\/.+\..+\/(\?PostData_v\d=)?%20(.+)%40/.test(content) ? content : ''
```

Comment:
does this break the backward compatibility?

Reply:
I have been tested on old posts. They are working fine.